Packs/Core/Playbooks/playbook-Netcat_Makes_or_Get_Connections.yml

```diff
@@ -0,0 +1,955 @@
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "18":
+    id: "18"
+    taskid: 85dafa8d-8c75-465f-8699-27d29a01682d
+    type: regular
+    task:
+      id: 85dafa8d-8c75-465f-8699-27d29a01682d
+      version: -1
+      name: Get destination IP reputation
+      description: Checks the specified IP address against the AbuseIP database.
+      script: '|||ip'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "15"
+    scriptarguments:
+      ip:
+        complex:
+          root: alert
+          accessor: remoteip
+          transformers:
+          - operator: uniq
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 730,
+          "y": 715
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "19":
+    id: "19"
+    taskid: 61055de3-bb9f-4984-8da3-45fdc40479ba
+    type: title
+    task:
+      id: 61055de3-bb9f-4984-8da3-45fdc40479ba
+      version: -1
+      name: Execute Remediation
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "41"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 450,
+          "y": 1945
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "20":
+    id: "20"
+    taskid: ccf44f37-71f0-4852-8606-640213d46253
+    type: condition
+    task:
+      id: ccf44f37-71f0-4852-8606-640213d46253
+      version: -1
+      name: Check if Domain Name Exist?
+      description: Checks if the domain name in the alert exists.
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "21"
+      "yes":
+      - "16"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: isNotEmpty
+          left:
+            value:
+              simple: alert.dnsqueryname
+            iscontext: true
+          right:
+            value: {}
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 140,
+          "y": 715
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "21":
+    id: "21"
+    taskid: 55b7d40a-0f6c-4ed8-8689-c7ec3c842dbd
+    type: regular
+    task:
+      id: 55b7d40a-0f6c-4ed8-8689-c7ec3c842dbd
+      version: -1
+      name: Get Commandline prevalence
```

Comment:
Can all the prevalence command be parallel?
+ Why we don't check for the domain prevalence as well?